blocks/extra-large-manual-promo-block/themes/news.json

```diff
@@ -3,12 +3,18 @@
 		"styles": {
 			"default": {
 				"components": {
+					"heading": {
+						"color": "var(--text-color)"
+					},
 					"link": {
 						"font-family": "inherit",
 						"font-size": "inherit",
 						"font-weight": "inherit",
 						"line-height": "inherit"
 					},
+					"overline": {
+						"color": "var(--text-color)"
+					},
```

Comment:
To go with the other comment, we can move
```
"overline-hover": {
  "color": "var(--text-color-subtle)"
}
```
here, and eliminate the `*-promo-overline` tokens.